src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain
+
+private const val MINIMUM_NUMBER_OF_CARS = 2
+
+class Cars(cars: List<Car>) {
+    val values: List<Car>
+
+    init {
+        require(cars.size >= MINIMUM_NUMBER_OF_CARS) { "자동차 경주는 적어도 ${MINIMUM_NUMBER_OF_CARS}대 이상의 자동차가 있어야 가능합니다." }
+        this.values = cars
+    }
+
+    fun move(canMove: () -> Boolean) {
+        values.forEach { car -> car.move(canMove) }
+    }
```

Comment:
```suggestion
class Cars(private val cars: List<Car>) {

    init {
        require(cars.size >= MINIMUM_NUMBER_OF_CARS) { "자동차 경주는 적어도 ${MINIMUM_NUMBER_OF_CARS}대 이상의 자동차가 있어야 가능합니다." }
    }

    fun move(canMove: () -> Boolean) {
        cars.forEach { car -> car.move(canMove) }
    }
```

자바와 다르게 코틀린은 바로 맵핑할 수가 있습니다.

Reply:
앗 이 부분은 약간 자바에서 static factory 메서드를 만드려고 하다가
되돌렸는데, 코드가 약간 애매하게 남아 있었네요!
변경해두었습니다 :)